modules/ROOT/pages/query-unified-schema.adoc

```diff
@@ -21,7 +25,70 @@ Choose one of the following options:
 ** *I already have credentials...*: Click *Next*, provide an existing client ID and secret pair, and then click *Authenticate & Request Access*.
 . To change your access method from the *Run Query* page, click *Manage Credentials* > *Switch credentials*.
 
-=== Request Access to Environments Using an External Client Provider
+
+== Write and Run a Query
+
+Use the query editor to write, run, and troubleshoot queries. 
+
+. If any of the APIs in the unified schema require pass-through authentication, click *Add HTTP Headers*, and:
+
+.. Select a header name.
+.. Enter a header value.
+.. To specify additional header name and value pairs for other APIs, click *+Add Header* x 
+.. Click *Save*.
+.. To edit the HTTP headers, click *Edit HTTP Headers*.
+
+. In the query editor, start typing your query using the in-line autocomplete function or type *Option/Alt + Shift + Space* to open the autocomplete menu.
++
+image::datagraph-qsg-query-autocompletion.png[Using inline autocompletion to write a query,70%]
+
+. Optionally, in the *Query Variables* pane, add any query variables you want to use.
++
+image::datagraph-query-variable.png[Inserting a query variable in the query editor,70%]
+
+. Click *Run*.
++
+image::datagraph-qsg-query-result.png[Query editor displays query results]
+. To see results with query tracing, click *Trace query* > *Run*.
++
+image::datagraph-qsg-query-traces.png[Query trace view open with query results]
++
+Query Trace results for Anypoint DataGraph provide the following information:
++
+* Time taken by Anypoint DataGraph to parse and validate the query
+* Total response time for the entire query
+* Duration of requests to each source API in the query
+
+. To see response logs associated with the query, click *View response logs*.
++
+image::datagraph-qsg-response-logs.png[Query response logs page]
++
+Log levels for Anypoint DataGraph include, DEBUG, INFO, WARN, and ERROR.
+. Click *View History* to access this same query—or others—at a later time.
```

Comment:
This is weird:
query—or others—at

I think Kim frowns upon dashes used like this, too. But, if you keep them, I think they need spaces.